
from django.core.management.base import CommandError


class ScopeException(CommandError):
    pass